_data/cves/2025/CVE-2025-5740.json

```
@@ -0,0 +1,162 @@
+{
+  "containers": {
+    "adp": [
+      {
+        "metrics": [
+          {
+            "other": {
+              "content": {
+                "id": "CVE-2025-5740",
+                "options": [
+                  {
+                    "Exploitation": "none"
+                  },
+                  {
+                    "Automatable": "no"
+                  },
+                  {
+                    "Technical Impact": "total"
+                  }
+                ],
+                "role": "CISA Coordinator",
+                "timestamp": "2025-06-10T13:08:59.880379Z",
+                "version": "2.0.3"
+              },
+              "type": "ssvc"
+            }
+          }
+        ],
+        "providerMetadata": {
+          "dateUpdated": "2025-06-10T13:09:05.605Z",
+          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
+          "shortName": "CISA-ADP"
+        },
+        "title": "CISA ADP Vulnrichment"
+      }
+    ],
+    "cna": {
+      "affected": [
+        {
+          "defaultStatus": "unaffected",
+          "product": "EVLink WallBox",
+          "vendor": "Schneider Electric",
+          "versions": [
+            {
+              "status": "affected",
+              "version": "All Versions"
+            }
+          ]
+        }
+      ],
+      "descriptions": [
+        {
+          "lang": "en",
+          "supportingMedia": [
+            {
+              "base64": false,
+              "type": "text/html",
+              "value": "CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') vulnerability exists that\ncould cause arbitrary file writes when an unauthenticated user on the web server manipulates file path. \n\n<br>"
```

```suggestion
              "value": "CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') vulnerability exists that\ncould cause arbitrary file writes when an authenticated user on the web server manipulates file path. \n\n<br>"
```
Volgens mij is dit een authenticated vulnerability, tenzij hier geen authorization op wordt uitgevoerd, maar dan zou dat nog een tweede vulnerability zijn.